lib/src/spotify_credentials.dart

```diff
@@ -6,8 +6,11 @@ part of spotify;
 class SpotifyApiCredentials {
   final String clientId;
   final String clientSecret;
+  final String refreshToken;
 
-  SpotifyApiCredentials(this.clientId, this.clientSecret);
+  SpotifyApiCredentials(this.clientId, this.clientSecret, [this.refreshToken = null]);
```

Comment:
Please remove that change from the PR, since it's coming in with #28 :smile_cat: